sdk/keyvault/azure-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/models/SecretProperties.java

```diff
@@ -231,9 +229,9 @@ public String getContentType() {
      * Set the contentType.
      *
      * @param contentType The contentType to set
-     * @return the SecretBase object itself.
+     * @return the SecretProperties object itself.
```

Comment:
@return the **updated** SecretProperties object.

Reply:
added